css/properties/position.json

```
@@ -257,14 +257,24 @@
               "opera_android": {
                 "version_added": "43"
               },
-              "safari": {
-                "prefix": "-webkit-",
-                "version_added": "6.1"
-              },
-              "safari_ios": {
-                "prefix": "-webkit-",
-                "version_added": "6.1"
-              },
+              "safari": [
+                {
+                  "prefix": "-webkit-",
+                  "version_added": "6.1"
+                },
+                {
+                  "version_added": "13"
+                }
+              ],
```

```suggestion
              "safari": [
                {
                  "version_added": "13"
                },
                {
                  "prefix": "-webkit-",
                  "version_added": "6.1"
                }
              ],
```
See https://github.com/mdn/browser-compat-data/blob/master/schemas/compat-data-schema.md#the-support_statement-object:
> sort such arrays with entries applying to the most recent browser releases first and sort entries with prefixes or flags after those without